nodejs/week3/index.js

```diff
@@ -0,0 +1,72 @@
+const { response } = require("express");
+const express = require("express");
+const app = express();
+app.use(express.urlencoded({ extended: true }));
+
+app.get("/calculator/:operator", (req, res,) => {
```

Comment:
Hmm I see what you tried to do with operator parameter here and the way you implemented is clean so I'm not going to suggest you to change it :) . However, it would be more simple to just have separate routes for each operation, eg
```
app.get("/calculator/addition", (req, res)
```